projects/online/online/utils/gdb.py

```diff
@@ -61,8 +66,12 @@ def submit_pe(
         self,
         result: bilby.core.result.Result,
         mollview_plot: plt.figure,
+        skymap: "table.Table",
         graceid: int,
     ):
+        skymap_fname = self.write_dir / "amplfi.fits"
```

Comment:
Am I missing where this file gets submitted to GraceDB? I saw it on one of the events, but I don't see how it happens

Reply:
couple commits from aframe account were mising. should be fixed